README.md

```diff
@@ -1,40 +1,40 @@
 # Microsoft Authentication Library (MSAL) for Python Preview
 
-The MSAL library for Python enables your app to access the
-[Microsoft Cloud](https://cloud.microsoft.com)
-by supporting authentication of users with
-[Microsoft Azure Active Directory accounts](https://azure.microsoft.com/en-us/services/active-directory/)
-and [Microsoft Accounts](https://account.microsoft.com) using industry standard OAuth2 and OpenID Connect.
-Soon MSAL Python will also support [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/).
-
-Not sure whether this is the SDK you are looking for? There are other Microsoft Identity SDKs
+
+| Reference Docs
+|---------------
+ [![readthedocs](https://readthedocs.org/projects/pip/badge/)](https://msal-python.readthedocs.io/)
+
+The Microsoft Authentication Library for Python enables applications to implement authentication flows using the [Microsoft identity platform](https://aka.ms/aaddevv2). It allows you to sign in users or apps with Microsoft identities ([Azure AD](https://azure.microsoft.com/services/active-directory/), [Microsoft Accounts](https://account.microsoft.com) and [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/) accounts) and obtain tokens to call Microsoft APIs such as [Microsoft Graph](https://graph.microsoft.io/) or your own APIs registered with the Microsoft identity platform. It is built using industry standard OAuth2 and OpenID Connect protocols
```

Comment:
to implement authentication flows  => to acquire security tokens to call protected Web APIs ? Not sure if implementing auth flow is very customer focused?

Reply:
Updated